src/content/docs/builders/tools/wallets/custodial/crossmint.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: Crossmint
+description: Intro to Wallet Crossmint
+---
+import ToolLayout from "../../../../../../components/Tool/ToolLayout.astro";
+import {Steps} from "@astrojs/starlight/components";
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+
+<ToolLayout
+    introParagraph="
+Crossmint enables you to create application-specific wallets and provides all necessary blockchain infrastructure for complete use cases, including fetching content, signing transactions, and transferring assets."
+    toolName="Crossmint"
+    toolLink="https://docs.crossmint.com/wallets/custodial-wallets/overview"
+    toolDocsLink="https://docs.crossmint.com/wallets/custodial-wallets/overview"
+>
+
+With Crossmint, you can seamlessly integrate web3 capabilities into your website or app. Create wallets for existing users via API and for new users at sign-up. Additionally, you can build quick, plug-and-play activations with no frontend work required.
```

Comment:
Proposal

Crossmint enables you to seamlessly integrate key Web3 functionalty into your application or game.
Create wallets invisibly through their simple APIs for new or existing users. You can use this to quickly enable blockchain without forcing seed phrases and passwords onto users.